test/xds/xds_server_integration_test.go

```diff
@@ -111,12 +111,11 @@ func setupGRPCServer(t *testing.T, bootstrapContents []byte) (net.Listener, func
 		},
 	}
 
-	server, err := xds.NewGRPCServer(grpc.Creds(creds), testModeChangeServerOption(t), xds.BootstrapContentsForTesting(bootstrapContents))
-	if err != nil {
+	sopts := []grpc.ServerOption{grpc.Creds(creds), testModeChangeServerOption(t), xds.BootstrapContentsForTesting(bootstrapContents)}
```

Comment:
same. no need to move if not being used anywhere else

Reply:
done